pkg/syncer/statussyncer.go

```diff
@@ -27,7 +27,14 @@ func deepEqualStatus(oldObj, newObj interface{}) bool {
 	return equality.Semantic.DeepEqual(oldStatus, newStatus)
 }
 
+const statusSyncerAgent = "kcp#status-syncer/v0.0.0"
```

Comment:
this is something I would like to see everywhere, for every controller and component we build. Is therre a good place to add this centrally? E.g. in the kcp startup code?

Reply:
Yes, but //TODO?

Reply:
yes, as convention